### PR TITLE
Add node.js market data subscription example over long-polling

### PR DESCRIPTION
**Pull request by ericlu88**
This PR adds an example for market data subscription over long-polling. This is addressing issue #76

### CODE REVIEW

**Comment by ericvw** [comment]
To make the example simple, and because it is an example, I would recommend that we use the synchronous versions of the `POST` and `GET` requests.  The async versions makes the example more complicated and detracts from the real purpose of what is being illustrated.

**Comment by ericlu88** [comment]
Refactored the example based on discussion.

File: examples/node/MarketDataSubscription_LongPoll.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    request.post({
+        url: url + '/subscription?action=start',
+        body: [
+                { security: 'AAPL US Equity', correlationId: 0, fields: ['LAST_PRICE'] },
+                { security: 'GOOG US Equity', correlationId: 1, fields: ['LAST_PRICE'] }
+              ],
+        json: true,
+        agentOptions: opt
+    }, function (error, response, body) {
+        // Subscribed
```
Comment (ericvw) [inline examples/node/MarketDataSubscription_LongPoll.js]: Should this be `console.log` similar to the `unsubscribe()`?

File: examples/node/MarketDataSubscription_LongPoll.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        console.log(body);
+        poll();
+    });
+}
+
+function poll()
+{
+    request.get({
+        url: url + '/subscription?pollid=' + counter,
```
Comment (ericvw) [inline examples/node/MarketDataSubscription_LongPoll.js]: nit: `util.format` may be easier to grok the concatenation here

File: examples/node/MarketDataSubscription_LongPoll.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+{
+    request.post({
+        url: url + '/subscription?action=start',
+        body: [
+                { security: 'AAPL US Equity', correlationId: 0, fields: ['LAST_PRICE'] },
+                { security: 'GOOG US Equity', correlationId: 1, fields: ['LAST_PRICE'] }
+              ],
+        json: true,
+        agentOptions: opt
+    }, function (error, response, body) {
```
Comment (ericvw) [inline examples/node/MarketDataSubscription_LongPoll.js:26]: What happens when `error` occurs?  Should we just log and bail?  Same comment for the other callback handlers in this file.

**Comment by ericvw** [comment]
lgtm - please squash

**Comment by ericlu88** [comment]
Squashed.

**Comment by ericvw** [comment]
TravisCI is experiencing issues today.  Since we know this won't affect `make check`, I am going to go ahead and merge anyways.

**Pull request merged**